aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1,54 +1,28 @@
 import { Anchor, Box, Card, Text } from 'grommet';
 import { StatusCritical, StatusInfo } from 'grommet-icons';
 
-Messaging within HPE applications should be informative, concise, and trustworthy.
-It should be delivered in active, positive, objective, conversational
-American English and written from a third-person [point of view](#point-of-view).
-Executed properly, messaging within HPE applications should leave users
-feeling and perceiving HPE and its products to be intelligent, experienced,
-inclusive, and trustworthy.
+The HPE Design System employs text-based content that cultivates an exceptional user experience.
+<!-- The above sentence will be the leading line and moved to the Voice and Tone card in Foundations once approved -->
 
-## Messaging Voice, Tone, and Content
+The user experience is built upon our content. Every word we write is a carefully-considered concept that voices our values: 
 
-Many definitions for tone and voice exist, however, reading through the
-various definitions, a common picture emerges:
+- **Open**: We foster inclusive experiences that are personal, accessible, and transparent.
+- **Courageous**: We empower users with decisive and experienced guidance.
+- **Inspired**: We share our drive of 'yes we can' with users.
 
-| Voice                         | Tone                                        |
-| ----------------------------- | ------------------------------------------- |
-| Who they hear                 | What they hear                              |
-| Personality                   | Attitude                                    |
-| How you are doing the writing | Who they hear                               |
-| Style                         | How your message comes across               |
-| What makes us uniquely us     | How we communicate to our specific audience |
+Refer to the guidance for [HPE's Voice](https://brandcentral.hpe.com/brand-central/content/voice) for more information.
 
-### Voice
+## Using Our Voice
 
-Voice is who the reader "hears" delivering messaging; embedded within it is the
-worldview of the author. Voice can be spoken, witty, friendly, formal, etc.
-At HPE, the desire is for HPE's voice to be perceived as intelligent, experienced,
-objective, inclusive, and trustworthy &mdash; supporting HPE's worldview which
-believes in accelerating what's next, making bold moves, the power of _yes we can_, and
-being a force for good.
+We exercise empathy in our voice through situational tones that speak to each context, audience, and subject matter. We can be a force for good by creating content that is:
```

Comment:
Can we be more consistent in how we reference the HPE values? For instance, we put quotes around "yes we can" above but we don't put quotes around "be a force for good" here.

Reply:
This is a fair point and the difference is subtle. For perspective, I'm using quotes in the first instance to separate 'yes we can' as its own phrase from the surrounding text. In the second instance, I'm embedding 'be a force for good' into the sentence as an instruction rather than a separate phrase. See how these two sound different when read aloud:
- We can be a force for good by creating reliable content.
- We can "be a force for good" by creating reliable content.

Would it make sense to italicize _be a force for good_ to emphasize it as a value?